game/magic/artifact/item.go

```diff
@@ -0,0 +1,286 @@
+package artifact
+
+import (
+	"bytes"
+	"fmt"
+	_ "log"
+
+	"github.com/kazzmir/master-of-magic/game/magic/data"
+	"github.com/kazzmir/master-of-magic/lib/lbx"
+)
+
+// TODO: use Artifact
+type Item struct {
+	Name string
+	Image int
+	Slot ArtifactSlot
```

Comment:
I guess the slot isn't really needed since the type implies the slot. I'm not really sure why the lbx file has it.